### PR TITLE
Start screen and intro fonts

### PR DESCRIPTION
**Pull request by instMeyFTW**

### CODE REVIEW

File: dunwoody_disaster/views/StartMenu.py [hunk 1]
```diff
@@ -110,31 +112,51 @@ def __init__(self):
...
         layout.setSpacing(0)
         layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(layout)
 
         self.movie = QMovie(DD.ASSETS["FinalTitle"])
 
+        row = 0
+
+        # layout.addItem(
+        #     QSpacerItem(0, 50, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 0)
```
Comment (NoahBartell) [inline dunwoody_disaster/views/StartMenu.py:124]: Comment

File: dunwoody_disaster/views/StartMenu.py [hunk 2]
```diff
@@ -110,31 +112,51 @@ def __init__(self):
...
+
+        # btns_Lyt.addItem(
+        #     QSpacerItem(50, 0, QSizePolicy.Fixed, QSizePolicy.Fixed))
 
-        close = QPushButton("Exit")
-        close.setStyleSheet(btn_style)
-        close.clicked.connect(self.exitGame)
-        btns.addWidget(close)
+        # layout.addItem(
+        #     QSpacerItem(0, 50, QSizePolicy.Fixed, QSizePolicy.Fixed), 4, 0)
```
Comment (NoahBartell) [inline dunwoody_disaster/views/StartMenu.py:164]: comments

File: dunwoody_disaster/views/StartMenu.py [hunk 3]
```diff
@@ -110,31 +112,51 @@ def __init__(self):
...
+        #     QSpacerItem(0, 50, QSizePolicy.Fixed, QSizePolicy.Fixed), row, 0)
+
         bkg = MovieLabel()
         bkg.setMovie(self.movie)
         bkg.setAlignment(Qt.AlignmentFlag.AlignCenter)
-        layout.addWidget(bkg)
+        layout.addWidget(bkg, row, 0)
+
+        # layout.addItem(
+        #     QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.MinimumExpanding), 2, 0)
```
Comment (NoahBartell) [inline dunwoody_disaster/views/StartMenu.py:132]: Comment

**Review by NoahBartell — APPROVED** [review]
Besides some clean up with old comments Lgtm

**Review by chanson02 — APPROVED** [review]
I guess it's okay.

**Pull request merged**